app/controllers/checkout_controller.rb

```diff
@@ -96,6 +96,9 @@ def before_payment
   def checkout_workflow(shipping_method_id)
     while @order.state != "complete"
       if @order.state == "payment"
+        @order.updater.update_totals
+        @order.updater.update_pending_payment
```

Comment:
I agree with Rubocop that these lines add complexity to an already complex method.

Reply:
I pulled it out to a named method for now. We'll be deleting this whole controller when Split Checkout is merged, so it feels temporary anyway :man_shrugging: